new/PRS13975BadaMaryam.xml

```diff
@@ -49,13 +49,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Bāʿda Māryām</persName>
                     <name role="qesa gabaz">ቂሰ፡ ገበዝ፡</name>
                     <faith type="EOTC"/>
-                    <occupation type="ecclesiastic">Senior ecclesiastic (qesa gabaz) of the church of 
-                        <placeName ref="LOC5843">Tadbaba Maryam</placeName> in the time of abbot mamher 
-                        <persName ref="PRS0000Maqaryos">Maqaryos</persName>.
-                            </occupation>
+                    <occupation type="ecclesiastic"></occupation>
                     <birth/>
                     <death/>
-                    <floruit when="1728"/>
+                    <floruit when="1728">Senior ecclesiastic (qesa gabaz) of the church of 
+                        <placeName ref="LOC5843">Tadbaba Maryam</placeName> in the time of abbot mamher 
```

Comment:
please provide the exact complete ID for Tadbaba Maryam

please transcribe mamher